wolfcrypt/src/aes.c

```diff
@@ -1103,7 +1124,7 @@ static void wc_AesEncrypt(Aes* aes, const byte* inBlock, byte* outBlock)
             printf("sz = %d\n", AES_BLOCK_SIZE);
         #endif
 
-        /* check alignment, decrypt doesn't need alignment */
+        /* check alignment, encrypt doesn't need alignment */
```

Comment:
This is the encrypt function and alignment is needed for encrypt.  The comment is supposed to help someone who wonders "if encrypt is doing alignment checking why isn't decrypt"?  Decrypt uses an aligned temp.